src/main/java/mate/academy/util/HibernateUtil.java

```diff
@@ -6,9 +6,6 @@
 public class HibernateUtil {
     private static final SessionFactory sessionFactory = initSessionFactory();
 
-    private HibernateUtil() {
-    }
-
```

Comment:
why do you remove this constructor?

Reply:
I don't know, I'm confused)

Reply:
The SessionFactory is a very large object, so we use the Singleton pattern, so that we have only one such object in our program. And so we created a private constructor in the HibernateUtil class so that no one accidentally creates another instance of SessionFactory.